pallets/nft/src/tests.rs

```diff
@@ -69,13 +74,17 @@ fn test_issue_and_claim_eth() {
 			ClassType::Claim(merkle_root),
 		));
 
-		// assert_eq!(
-		// 	events(),
-		// 	[Event::Nft(crate::Event::CreatedClass(
-		// 		alice_account.clone(),
-		// 		0
-		// 	)),]
-		// );
+		// even we clear events before create_class, as Pot address never exists and balance on it being added, it will always trigger system event and balance event first time
+		// Improvement will be made in full test - issue #71
+		assert_eq!(
+			events(),
```

Comment:
As we have discussed, we should use index instead of compare the whole event array

Reply:
OK. I will rearrange this part into this PR.